storage/gc_queue.go

```diff
@@ -328,18 +328,18 @@ func (gcq *gcQueue) lookupGCPolicy(rng *Replica) (config.GCPolicy, error) {
 		return config.GCPolicy{}, util.Errorf("gossiped info is not a prefix configuration map: %+v", info)
 	}
 
-	// Verify that the range doesn't cross over the zone config prefix.
-	// This could be the case if the zone config is new and the range
+	// Verify that the replica doesn't cross over the zone config prefix.
```

Comment:
Updated this line, and a few other places where the replica's range is the object of interest.